### PR TITLE
fix(IntextCitation): styles and arrow updated EHF-13936 EHF-13937

### PR DESCRIPTION
**Pull request by yasodaesarapu**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ class InTextCitation extends React.PureComponent {
         rel={isNoFollow ? 'nofollow' : ''}
       >
         View Source
-        <ArrowDirectional direction="right" strokeWidth="3" transform="rotate(-45)" />
+        <Img src="https://images.everydayhealth.com/react-component-images/right-up-arrow.svg" alt="right up arrow" className="in-text-citation__lightbox-arrow" resizeOptions={resizeOptions} />
```
Comment (MiSTERSPOONS) [inline src/index.js:83]: maybe put this as a constant or a defaultProp?

**Review by MiSTERSPOONS — APPROVED** [review]

File: src/index.js [hunk 2]
```diff
@@ -162,7 +162,7 @@ class InTextCitation extends React.PureComponent {
       onMouseEnter={this.toggleMouseEventLightBox}
       onMouseLeave={this.toggleMouseEventLightBox}
     >
-      <ArrowDirectional direction="right" strokeWidth="3" transform="rotate(-45)" />
+      <Img src="https://images.everydayhealth.com/react-component-images/right-up-arrow.svg" alt="right up arrow" className="in-text-citation__lightbox-arrow" resizeOptions={resizeOptions} />
```
Comment (MiSTERSPOONS) [inline src/index.js:165]: same here

File: src/index.js (hunk 1 above)
Comment (yasodaesarapu) [reply src/index.js]: @MiSTERSPOONS added as constant

File: src/index.js (hunk 2 above)
Comment (yasodaesarapu) [reply src/index.js]: @MiSTERSPOONS added as constant

**Pull request merged**